MonoGame.Framework/GameWindow.cs

```diff
@@ -37,7 +37,8 @@ public abstract class GameWindow {
 
 		public abstract string ScreenDeviceName { get; }
 
-		private string _title;
+		private string _title = MonoGame.Utilities.AssemblyHelper.GetDefaultWindowTitle();
```

Comment:
It's better to move assigning of variables to the constructor, it's more readable to be able to see all the starting values in there instead of searching for their declarations.